internal/activePlot.go

```diff
@@ -150,8 +152,29 @@ func (ap *ActivePlot) RunPlot() {
 	if ap.Threads > 0 {
 		args = append(args, fmt.Sprintf("-r%d", ap.Threads))
 	}
+	if ap.PlotSize == 32 {
+		args = append(args, "-k32")
+	} else {
+		args = append(args, fmt.Sprintf("-k%d", ap.PlotSize))
+	}
 	if ap.Buffers > 0 {
 		args = append(args, fmt.Sprintf("-b%d", ap.Buffers))
+	} else {
```

Comment:
blocksize should be a separate config.  if not set then use the chia default

Reply:
i tried not setting this, but chia seems default 3389 and won't change with k-size

chia plots create -k 32 -t /home/dlo/Downloads/test -d /home/dlo/Downloads/test
chia plots create -k 33 -t /home/dlo/Downloads/test -d /home/dlo/Downloads/test

both run in 3389 Buffer by default